dataactvalidator/config/sqlrules/d1_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+    row_number,
+    record_type,
+    fain
+FROM detached_award_financial_assistance as dafa
+WHERE dafa.submission_id = {0}
+AND dafa.record_type = 2
```

Comment:
I was actually thinking about it but in other sql rules the AND statements for the WHERE aren't indented (see a1, a2, etc). Do we want to start doing it?